### PR TITLE
telegraml-dashboard.0.1.0 - via opam-publish

### PR DESCRIPTION
**Pull request by nv-vn**
Dashboard-generating extension for TelegraML
A set of extensions for TelegraML to let the user
generate a custom web dashboard, including features
such as leaving groups, toggling commands, and
viewing the bot's status.

---
* Homepage: https://github.com/nv-vn/telegraml-dashboard
* Source repo: https://github.com/nv-vn/telegraml-dashboard.git
* Bug tracker: https://github.com/nv-vn/telegraml-dashboard/issues

---

Pull-request generated by opam-publish v0.3.1

### CODE REVIEW

File: packages/telegraml-dashboard/telegraml-dashboard.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This file was auto-generated by `ogen`.
+# Do not edit this by hand, as all changes will be
+# reverted upon the next run of `ogen`.
+
+opam-version: "1.2"
+maintainer: "nv-vn <nv@cock.li>"
+authors: "nv-vn <nv@cock.li>"
+homepage: "https://github.com/nv-vn/telegraml-dashboard"
+bug-reports: "https://github.com/nv-vn/telegraml-dashboard/issues"
+license: "GPL"
```
Comment (dsheets) [inline packages/telegraml-dashboard/telegraml-dashboard.0.1.0/opam]: `GPL-2.0` or `GPL-3.0`?

File: packages/telegraml-dashboard/telegraml-dashboard.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "telegraml-dashboard"]
+depends: [
+  "ocamlfind" {build}
+  "oasis" {build}
+  "unix"
```
Comment (dsheets) [inline packages/telegraml-dashboard/telegraml-dashboard.0.1.0/opam]: This should be `base-unix`.

**Comment by nv-vn** [comment]
Made those changes, not sure if anything else is broken (which would be quite likely). I'll check back in a few hours to see if it's broken.

**Comment by nv-vn** [comment]
Hm, looks like it's with gensqlite again. I've never been able to get anything to compile against the library without compiling it myself, so I'll have to investigate it some more before I fix this.

**Comment by samoht** [comment]
Can you try to rebase to master? We have upgraded the Ubuntu distribution which is running the tests recently.

**Comment by nv-vn** [comment]
@samoht I found the problem (the version of Gensqlite on OPAM wasn't up to date), but since I'm not the creator of the package there's not really much I can do about it at the moment (other than going back and changing the code to work with a different version/different library). I'll close the issue for now since I won't really get a chance to do anything of the sort for the next few weeks, and it's honestly not a high priority thing for me at the moment.

**Pull request closed without merging**